pkg/typesystem/typesystem_test.go

```diff
@@ -3242,11 +3241,10 @@ func TestTTUResolvesExclusivelyToDirectlyAssignable(t *testing.T) {
 								define parent: [group]
 								define viewer: member from parent
 					`,
-			objectType:               "folder",
-			tuplesetRelation:         "parent",
-			computedRelation:         "member",
-			expectDirectlyAssignable: true,
-			expectError:              false,
+			objectType:        "folder",
+			computedRelation:  "viewer",
+			userType:          "user",
+			expectCanFastPath: true,
```

Comment:
The lookup doesn't use the tupleset relation, instead it uses the object type, the target relation and the user type.